src/mpid/ch4/src/ch4_coll_impl.h

```diff
@@ -21,6 +21,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_Barrier_intra_composition_alpha(MPIR_Comm * c
                                                                    * ch4_algo_parameters_container)
 {
     int mpi_errno = MPI_SUCCESS;
+    int mpi_errno_ret = MPI_SUCCESS;
```

Comment:
Not a big fan of the naming here. Maybe `coll_err` or something else?

Reply:
Agreed. I've renamed this.